pcap_ex.c

```diff
@@ -290,7 +290,7 @@ pcap_ex_compile_nopcap(int snaplen, int dlt, struct bpf_program *fp, char *str,
 	char ebuf[PCAP_ERRBUF_SIZE];
 	int ret = -1;
 	
-	_mktemp(path);
+	mktemp(path);
```

Comment:
Yes, that's exactly what I did - replaced Windows-ism with the crossplatform code. Surely, the library should work on all supported platforms.